iModelCore/ecobjects/test/NonPublished/CustomAttributeTests.cpp

```diff
@@ -613,7 +613,7 @@ TEST_F (CustomAttributeTest, SerializeSchemaToXmlUtfString)
     caClass->CreatePrimitiveProperty (stringProp, caUtf8StringPropName, PRIMITIVETYPE_String);
 
         // *** WIP_PORTABILITY: Use an escape such as \u here. Don't try to use extended ascii directly
-    WCharCP caPropValueString = L"����ᳵ";
+    WCharCP caPropValueString = L"����ᳵ";

```

Comment:
was this on purpose? Looks to me like the unicode in the file broke?

Reply:
It appears to be broken, and I didn't tamper with this section of the code.

Reply:
Can you try and revert it? In this comment view, this looks like nothing changed, but actually, the characters changed from german special chars into some invalid stuff.
![chars](https://user-images.githubusercontent.com/18402661/224290718-7d11db4a-95f4-4e7f-8582-3896bfc094fe.png)